src/webparts/todolist/components/ToDoList.module.scss

```diff
@@ -19,6 +9,21 @@
 }
 
 .statusPending {
-  background-color: "#ffe092";
-  color: "#997825";
+  background-color: "[theme: warning, default:#fff4ce]";
+  color: "[theme: OrangeYellow20, default:#986f0b]";
```

Comment:
Not sure that OrangeYellow20 is correct token name. Most popular colors you can find here.
https://developer.microsoft.com/en-us/fluentui#/styles/web/colors/theme-slots

<img width="723" alt="image" src="https://user-images.githubusercontent.com/32543592/204173014-be557d76-6543-48ce-97b6-40f083ffe1c0.png">

Also you can open console at any sharepoint site and invoke this expression `__themeState__.theme` then you will get theme object